config/packages/tabler.yaml

```diff
@@ -38,7 +38,6 @@ tabler:
         tabler_registration: registration
         tabler_registration_register: registration_register
         tabler_password_reset: resetting_request
-        tabler_password_reset_sent: resetting_send_email
```

Comment:
why remove it? it is used here:
https://github.com/kevinpapst/TablerBundle/blob/main/templates/security/password-reset.html.twig#L10

Reply:
`password-reset.html.twig` will no longer exist.
`action` attr for the form is defined by the controller it self so only `tabler_password_reset` is required now.